.buildkite/pipelines/quality-gates/pipeline.tests-qa.yaml

```diff
@@ -3,37 +3,54 @@
 # this pipeline build will prevent further progression to the subsequent stage.
 
 steps:
-  - label: ":pipeline::kibana::seedling: Trigger Kibana Tests for ${ENVIRONMENT}"
-    command: echo "replace me with Kibana specific tests"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/basic-buildkite-agent:1688566364"
+  - label: ":pipeline::kibana::seedling: Trigger Kibana Serverless Tests for ${ENVIRONMENT}"
+    trigger: appex-qa-kibana-serverless-ftr-tests # https://buildkite.com/elastic/appex-qa-kibana-serverless-ftr-tests
+    build:
+      env:
+        ENVIRONMENT: ${ENVIRONMENT}
+      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-qa.yaml)"
 
-  - label: ":pipeline::fleet::seedling: Trigger Fleet Kibana Tests for ${ENVIRONMENT}"
-    command: echo "replace me with Fleet specific Kibana tests"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/basic-buildkite-agent:1688566364"
+  - label: ":pipeline::weight_lifter::seedling: Trigger RAC Alert Load tests for ${ENVIRONMENT}"
+    trigger: appex-qa-rac-alert-load # https://buildkite.com/elastic/appex-qa-rac-alert-load
+    build:
+      env:
+        ENVIRONMENT: ${ENVIRONMENT}
+        SERVERLESS_ONLY: true
+      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-qa.yaml)"
 
-  - label: ":pipeline::lock::seedling: Trigger Security Kibana Tests for ${ENVIRONMENT}"
-    command: echo "replace me with Security specific Kibana tests"
-    agents:
-      image: "docker.elastic.co/ci-agent-images/basic-buildkite-agent:1688566364"
+  - group: ":female-detective: Security Solution Tests"
+    key: "security"
+    steps:
+      - label: ":pipeline::female-detective::seedling: Trigger Security Solution quality gate script"
+        command: .buildkite/scripts/pipelines/security_solution_quality_gate/pipeline.sh
```

Comment:
This looks like a placeholder file for now, but we may have to follow the trigger pattern in the future based on the pre-command/post-command skips.

Reply:
Yeah, I agree, having external pipelines are probably a lot better for us, especially because that can run in a Kibana-managed agent. Once the team finds time to work on these tests, we could move it to a separate pipeline.